base/unicode/properties.jl

```diff
@@ -0,0 +1,157 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# Unicode properties, such as General Category
+# Unix/C is* convenience functions (for now)
+
+# whether codepoints are valid Unicode scalar values, i.e. 0-0xd7ff, 0xe000-0x10ffff
+isvalid(::Type{Char}, ch::Unsigned) = !((ch - 0xd800 < 0x800) | (ch > 0x10ffff))
+isvalid(::Type{Char}, ch::Integer)  = isvalid(Char, Unsigned(ch))
+isvalid(::Type{Char}, ch::Char)     = isvalid(Char, UInt32(ch))
+
+isvalid(ch::Char) = isvalid(Char, ch)
+
+# Unicode General Category constants
+
+"""Unicode character properties"""
+abstract UnicodeProperty
+"""Unicode character categories"""
+abstract CharCategory   <: UnicodeProperty
+
+"""Unicode letter character category"""
```

Comment:
Missing uppercase L. It would also be clearer if you added some kind of quote around the names of the categories.